### ISSUE TITLE
[Android] Garbage on screen on Adreno GPU

### ISSUE THREAD

**Issue by Dammirt** [FWGS/xash3d-fwgs#1673]
![Screenshot_20240512_202351_su xash engine test](https://github.com/FWGS/xash3d-fwgs/assets/169608206/a8079d1a-31a2-4c1c-8fd3-5d92816507bc)
This... thing appears on snapdragon 7 gen 1, Model BTK-W09

**Comment by a1batross** [comment]
Yeah, it's a known bug that happens during change level because we're not updating the screen with anything, and it gets filled with garbage by Adreno's famous tile renderer.

There was a report on this and this is probably a duplicate, but I can't find it, so I won't close it yet.

Maybe somebody smart will suggest a possible solution.

**Comment by Vladislav4KZ** [comment]
On Mali GPUs, a black screen is displayed instead of this garbage (Currently, I no longer have a device with a Mali GPU).
Here's the issue https://github.com/FWGS/xash3d-fwgs/issues/641

**Comment by WaspPC** [comment]
That happens with Delta Touch, Quad Touch, Zeta Touch and this emulator between loading screen, it does not bother me much, as long as the rest of the game plays great, then its ok for the loading screen.

**Comment by a1batross** [comment]
Please, stop this "emulator" nonsense. And wherever you heard it, tell them
these words.

Xash is NOT an emulator, there is NO emulator inside this project and there
NEVER was. It's a game engine that's compatible with, Half-Life 1 engine,
GoldSource. A bit more due to extensions, but no less.

**Comment by Vladislav4KZ** [comment]
This bug should be fixed after this commit: https://github.com/FWGS/xash3d-fwgs/commit/0da7ffe45f146670612af066386161b0e77f5973

I tested on a device with Adreno GPU.

**Comment by Vladislav4KZ** [comment]
Well, yes, this garbage can still appear for one frame during changelevel.

It's barely noticeable, but it's still there.

**Comment by a1batross** [comment]
Now fixed completely with 56ba2325

**Comment by Vladislav4KZ** [comment]
I just installed the latest build of the engine. The bug with garbage in one frame still remains.

**Comment by Vladislav4KZ** [comment]
Here is the engine build:
![Screenshot_20240711_114307_Xash3D FWGS (Test).png](https://github.com/FWGS/xash3d-fwgs/assets/22411953/59e7835f-2528-4e73-9b41-3f8e38befb4c)

**Comment by Vladislav4KZ** [comment]
https://github.com/FWGS/xash3d-fwgs/assets/22411953/2ec6ca20-0367-423d-914f-20ce1b2318cb
